### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by amilton33** [darthcloud/BlueRetro#727] [General]
I would like to ask a question, pin 9 (ACK) is used in the blueretro project, I am doing a project where an arduino nano converts the buttons from the PSP to the PS2, and this pin is not used in the code, just the data, command, attention and clock, is it mandatory to use it for blueretro? What would be the function of this pin?

**Comment by ghost** [comment]
Yes it is used on the blueretro, the playstation will consider the controller missing if the ack signal doesn’t come within 100us. 
I dont know what is the code you are using, but are you sure it really doesn't use the ack pin ? Have you tested it yet ?

**Reply by amilton33** [reply]
Yes, the code I'm using only defines 4 pins, command, attention, date and CLK, I'll do the tests yet, but I came up with this doubt, but seeing other arduino projects with communication with ps2 controls, they do not use ACK.
